MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/EntityId.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model\SearchCriteria\CollectionProcessor\FilterProcessor;
+
+use Magento\Framework\Api\Filter;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessor\FilterProcessor\CustomFilterInterface;
+use Magento\Framework\Data\Collection\AbstractDb;
+
+/**
+ * Class EntityId
+ * Used to filter the collection based on the filter selection.
+ * @package Magento\MediaGalleryUi\Model\SearchCriteria\CollectionProcessor\FilterProcessor
```

Comment:
Please remove the `Class EntityId` and `@package` lines from the class comments (applicable to all classes)